core/src/test/kotlin/fr/sncf/osrd/utils/DummyInfra.kt

```diff
@@ -112,36 +103,36 @@ class DummyInfra : RawInfra, BlockInfra {
     }
 
     override val physicalSignals: StaticIdxSpace<PhysicalSignal>
-        get() = TODO("Not yet implemented")
+        get() = StaticIdxSpace(blockPool.size.toUInt())
     override val logicalSignals: StaticIdxSpace<LogicalSignal>
-        get() = TODO("Not yet implemented")
+        get() = StaticIdxSpace(blockPool.size.toUInt())
 
     override fun getLogicalSignals(signal: PhysicalSignalId): StaticIdxList<LogicalSignal> {
-        TODO("Not yet implemented")
+        return makeIndexList(signal)
     }
 
     override fun getPhysicalSignal(signal: LogicalSignalId): PhysicalSignalId {
-        TODO("Not yet implemented")
+        return convertId(signal)
     }
 
     override fun getPhysicalSignalName(signal: PhysicalSignalId): String? {
         TODO("Not yet implemented")
     }
 
     override fun getSignalSightDistance(signal: PhysicalSignalId): Distance {
-        TODO("Not yet implemented")
+        return 400.meters
     }
 
     override fun getSignalingSystemId(signal: LogicalSignalId): String {
-        TODO("Not yet implemented")
+        return "BAL"
     }
 
     override fun getRawSettings(signal: LogicalSignalId): Map<String, String> {
-        TODO("Not yet implemented")
+        return mapOf("Nf" to "false")
     }
 
     override fun getNextSignalingSystemIds(signal: LogicalSignalId): List<String> {
-        TODO("Not yet implemented")
+        return listOf("BAL")
```

Comment:
Same as above, this should ideally not be static when we have different signaling types